api/v1/flow/flow.proto

```diff
@@ -12,8 +12,11 @@ option go_package = "github.com/cilium/cilium/api/v1/flow";
 
 message Flow {
     google.protobuf.Timestamp time = 1;
-    Verdict verdict = 2;
 
+    // uuid is a universally unique identifier for this flow.
```

Comment:
```suggestion
    // uuid is a universally unique identifier (version 4) for this flow.
```

Reply:
I would prefer to avoid commiting to v4 for now, we might want to revisit at some point and I don't think API consumers will rely on UUIDv4 since it has no guarantee (e.g. unlike UUIDv1 wrt ordering).

Reply:
I agree with @kaworu. Perhaps we should document that the ID is opaque, so no one makes any assumptions.

Reply:
@chancez after [Sebastian's comment](https://github.com/cilium/cilium/pull/23638#discussion_r1101610324) and naming the field `uuid` we're promising the field to be a UUID, so it's not opaque, but I don't think API consumers should rely on the UUID version.